portfolio/src/main/webapp/leave-comment/leave-comment.css

```diff
@@ -1,9 +1,17 @@
-#leave-comment-form {
+#leave-comment {
   min-width: 300px;
   max-width: 30%;
   left: 20%;
   position: fixed;
-  top: 30%;
+  top: 20%;
+}
+
+#greeting {
+  margin-down: 10px;
+}
+
+#show-on-sign-in {
+  visibility: hidden;
```

Comment:
On my browser the buttons get to be visible for a short while for the animation and then they disappear...

Reply:
Done.